docs/guides/tutorials/prefarm-audit.md

```diff
@@ -0,0 +1,121 @@
+
+Other relevant documents:
+
+- [Flow chart](/img/chia-custody-tool.png) to visualize how the custody tool works
+- [User guide](/guides/custody-tool-user-guide) to help you get up and running
+- [CLI reference](/custody-tool) for all custody commands used in this tutorial
+- [Prefarm Alert Tool](https://github.com/Chia-Network/prefarm-alert) for accessing the public prefarm config files
+
+## Prefarm Audit with the Internal Custody Tool
+The following steps assume that you have installed and setup the custody tool following the [User guide](/guides/custody-tool-user-guide). 
+The [chia-dev-tools](https://github.com/Chia-Network/chia-dev-tools#install) are also needed for this audit, examples shown have chia dev tools installed in a separate venv as the custody tool.
+
+1. Download the prefarm configuration files from the [Prefarm Alert Tool](https://github.com/Chia-Network/prefarm-alert/tree/main/singleton-metadata).
+2. Run the custody tool command `cic sync -c <config-file-path>.txt -db <unique_db_path>.sqlite`
+```bash
+(venv) cic sync -c .\prefarm_configs\cold-us-public-config.txt -db .\sync_cold_us.sqlite
+```
+
+:::info
+
+NOTE - this command will have no result if completed successfully but you will see the corresponding sqlite file created.
+
+:::
+
+3. Show details of the associated singleton by running `cic show -c -db <unique_db_path>.sqlite`
+```bash
+(venv) cic show -c -db ./sync_cold_us.sqlite
+
+Current time: 1696970325 (10/10/2023, 13:38:45)
+
+Config up to date: True
+
+Singleton:
+  - launcher ID: 6c77dce3c3bab525dab7883e8ad513a8f3ff127e872009b12836cbb1c8f26647
+  - amount left: 0
+  - amount to claim: 9187500100033000911
+
+Outstanding events:
+  PAYMENTS:
+  REKEYS:
+
+Config:
+ - current root: 9ab73274c5ca6cd0250b7b6b0352ad3190593bdb8f312d8aff3636c95208b0fb
+ - withdrawal timelock: 2592000 seconds
+ - payment clawback period: 7776000 seconds
+ - rekey cancellation period: 5184000 seconds
+ - rekey penalty: 1296000 seconds
+ - slow rekey timelock: 3888000 seconds
+```
+
+:::info
+
+This command will show the singleton launcher ID and the parameters associated with this specific prefarm wallet configuration (withdrawal, clawback, and cancellation periods).
+
+:::
+
+4. Show the singleton p2 address by running `cic p2_address -db <unique_db_path>.sqlite`
+```bash
+cic p2_address -db .\sync_cold_us.sqlite
+
+xch1jj0gm4ahhlu3ke0r0fx955v8axr6za7rzz6hc0y26lewa7zw6fws5nwvv6
+```
+---
+
+## Prefarm Audit with Block Records
+
+:::info
+
+NOTE - a high level of technical proficiency is needed to understand the details of this manual process for what the cic tool does above.
```

Comment:
Suggestion: 
Note: 
Use a colon instead of a hyphen. You can also use an em dash.